### PR TITLE
Add HCaptcha method

### PR DESCRIPTION
**Pull request by SmookeyDev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,25 @@ function are:
...
+  sitekey: 'the_key_extracted_from_the_page',
+  pageurl: 'https://www.site.with.hcaptcha/example',
+}).then(function(response) {
+  console.log(response.text);
+});
+
+>P0_eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJwYXNza2V5IjoiczhYam4vKzZnb...
+```
+
+`HCaptcha` is an async function. The parameters for `HCaptcha`
```
Comment (adrianodennanni) [inline README.md:132]: Function should be named `decodeHCaptcha`

**Review by adrianodennanni — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -116,6 +116,25 @@ function are:
 -   `action`: the action value used by the captcha.
 -   `enterprise`: optional (Boolean) switch for ReCaptcha Enterprise, default is `false`
 
+#### HCaptcha
```
[the remark this answers is not in the document]
Comment (SmookeyDev) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (SmookeyDev) [reply README.md]: fixed

**Comment by adrianodennanni** [comment]
Merged from dev

**Pull request closed without merging**